### PR TITLE
Update to lodash 4

### PR DESCRIPTION
**Pull request by njutte**
Update to use lodash 4 and build changes to support running on a server as well as in the browser.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,8 +5,22 @@
   "authors": [
     "Rob Hilgefort <rjhilgefort@gmail.com>"
   ],
+  "contributors": [
+    {
+    "name": "Ross Hadden",
```
Comment (rjhilgefort) [inline bower.json:10]: :+1:

File: docs/contributors.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
 
 * [rosshadden](https://github.com/rosshadden)
 * [AO16](https://github.com/AO16)
+* [njutte](https://github.com/njutte)
```
Comment (rjhilgefort) [inline docs/contributors.md:5]: :+1:

File: package.json [hunk 3]
```diff
@@ -2,10 +2,7 @@
   "name": "lodash-extras",
   "version": "0.0.10",
   "description": "An opinionated lodash mod with extras",
-  "main": "index.js",
-  "scripts": {
```
Comment (rjhilgefort) [inline package.json:6]: thanks for removing

File: package.json [hunk 4]
```diff
@@ -22,20 +22,42 @@
...
+      "github": "https://github.com/njutte"
+    }
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/rjhilgefort/lodash-extras/issues"
   },
   "homepage": "https://github.com/rjhilgefort/lodash-extras#readme",
   "devDependencies": {
-    "babelify": "^6.1.2",
```
Comment (rjhilgefort) [inline package.json:31]: :+1:

File: tests/tests.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (rjhilgefort) [inline tests/tests.js]: remove newline at top

Comment (njutte) [inline tests/tests.js]: Done

**Comment by rjhilgefort** [comment]
@njutte Thanks for the work!

**Pull request merged**